src/index.html

```diff
@@ -8,6 +8,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+
```

Comment:
Redundant empty line between parent and child elements

Reply:
I've deleted the line

Reply:
I've deleted the line